doc/source/releases/15.07_announce.rst

```diff
@@ -2,22 +2,60 @@
 July 2015 Galaxy Release (v 15.07)
 ===========================================================
 
+
 .. include:: _header.rst
 
+Highlights
+===========================================================
+
+**Interactive Environments**
+  The interactive environments (IE) framework features several updates this release
+  and Galaxy is now distributed with a new IE for RStudio implemented by Eric Rasche.
+
+**Workflow Editor Enhancements**
+  The workflow editor has been significantly revamped - it now uses newer tool form
+  components and allow construction of workflows with tools explicitly producing
+  output collections.
 
-Schedule
+**Reports Application Enhancements**
```

Comment:
Really? Ugh - is there a problem with my bootstrap script :(. I'll look into this.